docs/source/private-data/private-data.md

```diff
@@ -347,6 +348,346 @@ submitted a letter of credit, prior to the execution of the transfer chaincode.
 And instead of transactors directly hosting peers, they could transact through
 custodian organizations who are running peers.
 
+## Example scenario: Secured Private Asset transfer
+
+The private asset transfer smart contract [fabric sample](https://github.com/hyperledger/fabric-samples/tree/master/asset-transfer-secured-agreement/chaincode-go) 
```

Comment:
We have 2 example scenarios in here (which is fine) but one is at an h3 ### and the other is at at h2 ##.

Would it make more sense to have:
h2 ## Example scenarios
.. introduce and explain the difference between them and why we have included 2 here ....

h3 ### Example scenario 1: Asset transfer using private data collections
h3 ### Example scenario 2: Secured Private Asset transfer

Reply:
As suggested by others, converting to a separate tutorial.